src/test/java/io/confluent/connect/elasticsearch/integration/ElasticsearchConnectorBaseIT.java

```diff
@@ -146,13 +157,13 @@ protected void waitForRecords(int numRecords) throws InterruptedException {
 
   protected void writeRecords(int numRecords) {
     for (int i = 0; i < numRecords; i++) {
-      connect.kafka().produce(TOPIC, String.valueOf(i), String.format("{\"doc_num\":%d}", i));
+      connect.kafka().produce(TOPIC, String.valueOf(i), String.format("{\"doc_num\":%d,\"@timestamp\":%s}", i, "\"2021-04-28T11:11:22.000Z\""));
     }
```

Comment:
idk why i wrote it like this but lets change all of this to call `writeRecordsFromStartIndex(0, numRecords);`